chainntnfs/bitcoindnotify/bitcoind.go

```diff
@@ -640,17 +628,18 @@ func (b *BitcoindNotifier) RegisterSpendNtfn(outpoint *wire.OutPoint,
 	// outpoint/output script as spent.
 	//
 	// TODO(wilmer): use LoadFilter API instead.
-	if spendRequest.OutPoint == chainntnfs.ZeroOutPoint {
-		addr, err := spendRequest.PkScript.Address(b.chainParams)
+	if outpoint == nil || *outpoint == chainntnfs.ZeroOutPoint {
```

Comment:
Maybe convert the parameter to a value (no pointer) to prevent needing to deal with two values that have the same meaning? (non blocking)

Reply:
Sure, but I don't want to trigger even more changes since the ChainNotifier interface is used quite heavily throughout the codebase.

Reply:
Ok, next round :)